gdc_ng_models/models/batch.py

```diff
@@ -40,16 +40,24 @@ class Batch(Base, audit.AuditColumnsMixin):
     )
     name = sqlalchemy.Column(sqlalchemy.Text, nullable=False)
     project_id = sqlalchemy.Column(sqlalchemy.Text, nullable=False)
-    status = sqlalchemy.Column(
-        sqlalchemy.Enum("OPEN", "CLOSED", name="batch_status"),
-        default="OPEN",
-        nullable=False,
-    )
+    status = sqlalchemy.Column(sqlalchemy.Text, default="OPEN", nullable=False)
 
     members = orm.relationship(
         "BatchMembership", back_populates="batch", lazy="selectin"
     )
 
+    @orm.validates("status")
+    def validate_status(self, key, status):
+        if not status:
+            raise ValueError("status is required")
+
+        status = status.upper()
+
+        if status not in ["OPEN", "CLOSED"]:
```

Comment:
This is good. But for something that's kind of a constant, I'd save it as a constant like a `VALID_STATUSES` set at the top of the file rather than hiding it in a function.